lib/patreon/api.rb

```diff
@@ -33,16 +33,17 @@ def fetch_page_of_pledges(campaign_id, page_size, cursor=nil, includes=nil, fiel
     private
 
     def get_json(suffix)
-      http = Net::HTTP.new("api.patreon.com", 443)
+      http = Net::HTTP.new("www.patreon.com", 443)
       http.use_ssl = true
       http.verify_mode = OpenSSL::SSL::VERIFY_NONE
 
       #SECURITY HOLE
       http.set_debug_output($stdout) if ENV['DEBUG']
 
-      url = URI.parse("https://api.patreon.com/oauth2/api/#{suffix}")
+      url = URI.parse("https://www.patreon.com/api/oauth2/api/#{suffix}")
```

Comment:
I think here it would better to dispense with everything past the host name, so just the path

Reply:
I don't understand this comment.

Reply:
It looks we're doing this:

Reply:
Because we've already established the client?

Reply:
Yeah, the client above already has the hostname, ssl, and port. For an htt request, we don't need to repeat that detail